### ISSUE TITLE
[Suggestion] Tiny Things

### ISSUE THREAD

**Issue by accessdenied0** [MindBrain27/Sky-Greg#33]
Hey @MindBrain27 !

I so far REALLY like your modpack... its exactly something I always wanted: GregTech without much of the grind.
I just would have a few tiny things I would maybe love either as optional or general optimizations. 

Some are basically a bit from Nomnifactory, like a slightly less tedious recipe for the Blast furnace (like getting 2 Bricks per batch)
- adding Storage Drawers/Functional Storage anyway, since the Compression Drawers are quite handy...
- an Energy mod for mid-to-late game? (Maybe keeping an eye on Nuclearcraft: Neoteric)
but thats not as important...

The main reason why I did open this was the Lathe. I personally would love to see the "Harder Rods" to be deactivated personally: 
I think the Lathe in its current state is **almost** useless. There is no inzentive for me to build it in early game, since it only gives 1 Rod and 2 tiny dusts.
The thing is, I do have infinite resources basically and can just use a File... what makes the dusts more like of a hassle to deal with. Even things like the Steel dust, which might think could be useful, need the Blast-Furnace and 40! seconds to get it back into ingot form. Which is almost twice as long to make a new one with regular Iron and can't be melted. 
In addition at the MV tier there is the Extruder which allows to get 2 Rods from 1 Ingot, which takes a bit longer, but do not produce the byproduct. 

This is just my suggestion of course and I do understand if you like to keep it as close to the GT-NH - GT-Experience...

Another things I wanted to ask:
Why the Enchanter is disabled? I find to get like Silk-touch directly is quite handy... 

but that are just tiny suggestions,

I really like your Pack so far and I am curious where it goes!

**Comment by MindBrain27** [comment]
First of all thank you! Love to hear people are liking it! I'll answer these in a random order.

I do want to keep most things recipe related to GT as original as possible, however some configs are a little odd. I did have the harder dyes enabled at the start but it just kind of broke things for the most part.

I'm with you on the Lathe suggestion because tbh I'm in the IV age and I've only used one for Lenses. I think I'll revert that config to be normal. I used a File until an Extruder because it was faster and easier.

Storage Drawers will be coming but not Functional Storage as I personally like Storage Drawers more.

I do like the idea of a large energy mod opening up around IV age but it would need to be something like Nuclearcraft's scale. I just looked into that Neoteric mod and that could possibly do what I'm looking for but I'm going to play around with that one. I was playing OmniFactory before this so I do like the idea of large scale rf production to help but definitely not until EV or IV age. I like the idea of only GT energy for the first few ages.

The Enchanter got nuked with everything else in the mod except for Conduits, however I'm open to bringing that and maybe a few other blocks back. I just wanted to avoid rf usage with Ender IO for the main part but thinks like the Enchanter and Experience Obelisk can probably come back. I'm going to look through disabled items and see what makes sense.

**Comment by accessdenied0** [comment]
Cool I thank you for your quick response!
Also glad you liked some of the Ideas!

_I also remembered some potential suggestion, which I forgot to write down: 
I do like the REI and I was trying to combine it with EMI but it seem only to work with JEI:
Now [EMI](https://www.curseforge.com/minecraft/mc-mods/emi) can't pin like REI the recipe directly into the left panel, but it would have a lookup-/crafting history similar to the Nomnifactory-JEI-Utils one...

![image](https://github.com/MindBrain27/Sky-Greg/assets/8020013/750cc7ac-8593-4293-89a1-fa6f934448c5)

I am not sure which is better but I figured out, I might should mention this option. Sadly, REI seem to be not compatible with EMI, since it crashed in my tries._

**Edit: it seems to have currently an bug with GT, which might could get fixed, but some recipes might crash it... not yet figured out which.**

since it is possible to craft Spawner, if a mod which allows to move or even upgrade would be cool, but of course not necessary so far I see it... Apothesis would allow for instance the movement and upgrade of Spawner, and the items could be configured... like some circuits for control or similar. 

A minor thing I noticed is the Bender recipe, I know AE can handle tools, but it might be good if there is either an Alt-Recipe or some replacement for the Wrench...

Also an option would be to use the GT own Multiblock API to potential make a mid to late Energy Multiblock (like a Naquada Reactor (lategame) or something)

(Sorry for the wall of texts)
I am super excited to see what you have planned with this pack of yours!

**Comment by MindBrain27** [comment]
I definitely like the idea of the recipe history but for a few reasons I'd rather use EMI, in the future if GT fixes whatever bug it is that allows EMI and REI to be compatible then I would most likely add it, but I wouldn't be changing to JEI at this point. The recipe pin feature from REI is just too nice to get rid of in my opinion.

I'll look into the Bender recipe (and an alternative)because I do oddly remember that being like the only machine that needed a tool to craft for some reason and that's kind of strange.

I want to play around with custom multiblocks but I haven't gotten around to that quite yet. I'll probably add some multiblock machines and generators regardless of another power system.

More ideas the merrier, if I like it then I'll probably do it, if I don't then I'll at least explain why.

Should be an update tomorrow with some random changes and some Ender IO blocks coming back, the Enchanter, XP Vacuum, Vacuum Chest, and XP Obelisk. Also the Lathe change.

**Comment by MindBrain27** [comment]
Also the Spawner, I would love something like the Cardboard Box from Mekanism but I don't know of a mod that adds just that. I don't really want to add Apotheosis but if another mod added Spawner customization like they do I would definitely add that. I may go the Apotheosis route as it would allow more speed buffs and step height and stuff too. Apotheosis is a maybe.

**Comment by accessdenied0** [comment]
Yeah reasonable... and it sounds nice! 

As for the [Cardboard mod](https://www.curseforge.com/minecraft/mc-mods/sbm-cardboardboxes) is a single Block mod... The item is called "box" and it is a bit different to pick things up, but it does the same as the Cardboard Boxes from Meka.
The items it can be picked up can partial be allowed or disallowed. Even Spawners.

The only other mod I can think of, of Spawner changing is prob. Enhanced Mob Spawners, but it allows less "control". Soon you get an "Spawner key" you can adjust everything to your liking.

**Comment by MindBrain27** [comment]
I added the Cardboard mod and Apotheosis. I heavily changed most of Apotheosis recipes but also made it so you can craft all of the rarity materials and gem dust as well. I am playing in a mushroom forest biome so I'm basically playing in peaceful as mobs don't spawn there.

**Comment by MindBrain27** [comment]
Update today with all of this addressed, feel free to open another issue with more suggestions I am going to close this one for now.

**Comment by MindBrain27** [comment]
How do you enable the recipe lookup history with EMI? It turns out the AE2 encoding bug is due REI so I'm switching back to JEI/EMI but I can't seem to figure out how to enable to lookup history.

**Comment by MindBrain27** [comment]
nvm I figured it out.
